### DISCUSSION TITLE
Blob range requests don't work

### DISCUSSION THREAD

**Discussion by extracat** [vercel/community#5228] [Help] [answered]
### Summary

I've tried range request query to my blob as discribed here: 
https://vercel.com/docs/storage/vercel-blob#range-requests

That URL (from doc works), but my file URL doesn't work. It returns the entire file. What could be the problem?

### Steps to Reproduce

`curl -r 0-3 https://jcdgkezw1rsfhddh.public.blob.vercel-storage.com/pi_dec_1k.txt`

**Comment by swarnava** [comment]
Hi there,

This could be due to decimel value `.` present in your text file. Can you remove it and see if you are still facing this issue?

**Reply by extracat** [reply]
Did it. The same result.

**Comment by vvo** [comment, marked as answer]
We had an issue at some point around range requests, this is now working correctly:

```
> curl -r 0-3 https://jcdgkezw1rsfhddh.public.blob.vercel-storage.com/pi_dec_1k.txt
1415%
```
